app/views/layouts/application.html.erb

```diff
@@ -29,7 +29,7 @@
 
       <div>
         <%- if current_room && policy(current_room).edit? %>
-          <%= link_to t('icons.edit'), [:edit, current_space, current_room], class: 'no-underline', aria: { label: "Configure Room"} %>
+          <%= link_to t('icons.edit'), [:edit, current_space, current_room], class: 'no-underline', aria: { label: "Configure Section"} %>
```

Comment:
Maybe not for this PR, but it would be nice to pull this out to `config/locales/rooms/en.yml` and place it somewhere like `rooms.edit.link_to` (you'll want to figure out where we put the`edit.link_to` text in other spots)